magenta-lib/src/main/scala/magenta/tasks/UpdateCloudFormationTask.scala

```diff
@@ -53,6 +52,95 @@ trait RetryCloudFormationUpdate {
   }
 }
 
+class CloudFormationStackLookup(val strategy: CloudFormationStackLookupStrategy, val changeSetName: String, createStackIfAbsent: Boolean) {
```

Comment:
This no longer seems like a good name for this part of the data model as it isn't just about lookup. It contains the lookup strategy, the name of the change set to create and whether a stack should be created if it doesn't already exist. Perhaps the ever ambiguous `CloudFormationStackMetadata`??